apps/site/lib/site/shuttle_diversion.ex

```diff
@@ -68,8 +68,9 @@ defmodule Site.ShuttleDiversion do
       ]
 
       with %JsonApi{data: trips} <- Trips.by_route(trips_route, trips_params),
-           route_stops when is_list(route_stops) <- Stops.by_routes(true_route_ids, 0),
-           unique_trips = unique_trips_by_shape(trips) do
+           route_stops when is_list(route_stops) <- Stops.by_routes(true_route_ids, 0) do
```

Comment:
@digitalcora could you please rebase this and split it up so each change goes with the proper commit?

Reply:
I considered this, but thought an approach that resulted in cleaner changesets within individual commits would be better. If someone were looking at one of these commits in the future, it might be unclear which changes were related to the purpose of the commit, and which were Credo failures caused by changing the overall file, but having nothing to do with the code that was changed.

I can switch it if you think the other way is better; I'd add a body to each of the commits like "also addresses Credo failures on changed files".

IMO there's no _great_ way to handle this other than a mega-commit that just fixes all our Credo failures everywhere. (this was one nice thing Pronto actually did for us: it was somehow able to filter Credo failures to just the ones involving _lines_ that were changed in the PR)

Reply:
@digitalcora good point, let me think about it a minute